tests/legacy-cli/e2e/tests/build/chunk-optimizer.ts

```diff
@@ -0,0 +1,16 @@
+import assert from 'node:assert/strict';
+import { readFile } from 'node:fs/promises';
+import { execWithEnv } from '../../utils/process';
+
+/**
+ * AOT builds with chunk optimizer should contain generated component factories
+ */
+export default async function () {
+  await execWithEnv('ng', ['build', '--aot=true', '--configuration=development'], {
```

Comment:
Isn't chunk optimizer disabled in this? Since script optimisations are disabled during development builds?

https://github.com/angular/angular-cli/pull/27953/files#diff-725bf17eb0a0bed74c9aaca5f4c4ff2b2239a664a4baac4b29a43d961069adf7R70